docs/reference/generated/alert-dialog-root.json

```diff
@@ -12,16 +12,19 @@
       "description": "Whether the dialog is currently open."
     },
     "onOpenChange": {
-      "type": "(open, event, reason) => void",
+      "type": "((open: boolean, event: Event | undefined, reason: \"click\" | \"hover\" | \"focus\" | \"focus-out\" | \"escape-key\" | \"outside-press\" | \"trigger-press\" | undefined) => void)",
```

Comment:
We'll need to update the UI on the docs table, this is cut off, you need to scroll to read it,  I don't think it's the best experience (can be done in a follow up)

<img width="794" alt="Screenshot 2025-03-17 at 10 36 52" src="https://github.com/user-attachments/assets/3c14a139-ae38-484d-af26-631187592b24" />

Reply:
Yeah, I'm not a fan of the current layout.

Reply:
:laughing: Welcome to the never ending discussion about API page layout when the prop's types are getting more complex

Reply:
I wonder if the type are actually that valuable on the default UI.
Most of the widely used parameters can have their types deduced very easily from their name.

One could say that it's harder to read with the type because we have more characters.
Maybe we could try to stick with a minimalist typing on the default layout, and have a more detailed UI (tooltip ? accordion ? something else ?) that displays the full version.

Just thinking out loud :thinking:

Reply:
Is it possible to drop onto a new line at the `|`, the same way it's done in the `render` prop type on the Trigger component?

Reply:
There is a new line placed before each union member:

![image](https://github.com/user-attachments/assets/9f031302-a1fb-4969-afdb-c6fec668b4f2)

Doing this in function parameters would make it less readable IMO.